snap/snapcraft.yaml

```diff
@@ -47,15 +47,20 @@ parts:
       cd ${pkg_gopath}
 
       # build and install
+      arch=$(uname -m)
+      if [ $(arch)=="ppc64le" ]; then
```

Comment:
please use $arch, quotes and spaces

`if [ ${arch} == "ppc64le" ]; then`

Reply:
@devimc: Fixed.

Reply:
thanks